bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.11+ or Helm 3.0-beta3+
+- PV provisioner support in the underlying infrastructure
+- ReadWriteMany volumes for deployment scaling
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/kong
+```
+
+These commands deploy kong on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` statefulset:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release. Use the option `--purge` to delete all persistent volumes too.
+
+## Parameters
+
+The following tables list the configurable parameters of the kong chart and their default values.
+
+### Global Parameters
+| Parameter                              | Description                                                                                            | Default                                                 |
+|----------------------------------------|--------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `global.imageRegistry`                 | Global Docker image registry                                                                           | `nil`                                                   |
+| `global.imagePullSecrets`              | Global Docker registry secret names as an array                                                        | `[]` (does not add image pull secrets to deployed pods) |
+| `global.storageClass`     | Global storage class for dynamic provisioning                                                                  | `nil`                                                   |
+| `nameOverride`                         | String to partially override kong.fullname template with a string (will prepend the release name)   | `nil`                                                   |
+| `fullnameOverride`                     | String to fully override kong.fullname template with a string                                       | `nil`                                                   |
+| `clusterDomain`                      | Kubernetes cluster domain                                                                                                                                 | `cluster.local`                                         |
```

Comment:
These two parameters are not "global" parameters but "common" parameters instead:
```suggestion

### Common parameters

| Parameter | Description  | Default |
|----|----|----|
| `nameOverride` | String to partially override kong.fullname | `nil` |
| `fullnameOverride` | String to fully override kong.fullname | `nil` |
| `clusterDomain` | Kubernetes cluster domain  | `cluster.local` |
```

Reply:
Done